custom/inddex/reports/gaps_detail.py

```diff
@@ -89,4 +93,13 @@ def __init__(self, food_data):
 
     @property
     def rows(self):
-        return []
+        for row in _sort_rows(self._food_data.rows):
+            for gap_class, gap_code in [
+                    (ConvFactorGaps, row.conv_factor_gap_code),
+                    (FctGaps, row.fct_gap_code),
+            ]:
+                if gap_code != gap_class.AVAILABLE:
+                    manually_set = ['gap_type', 'gap_code', 'gap_desc']
+                    yield format_row([gap_class.name, gap_code, gap_class.get_description(gap_code)] + [
+                        getattr(row, col) for col in self.headers if col not in manually_set
+                    ])
```

Comment:
This block here will have 0, 1, or 2 rows for each row in `_food_data`, depending on what gaps are present.